### ISSUE TITLE
UI breaks for telescope when theme is dropdown.

### ISSUE THREAD

**Issue by MKrbm** [siduck76/NvChad#2281]
When I try to use `dropdown` theme for telescope, UI seems to break ( especially search bar is not center-aligned unlike default theme.

With `theme=dropdown`
![image](https://github.com/NvChad/NvChad/assets/57752203/3ac199c0-b164-484c-a2c8-55dbebfd6799)

without specifying `theme`
![image](https://github.com/NvChad/NvChad/assets/57752203/090ef5c0-614e-47e7-b353-8c946f1a1e2f)

How can I fix this?
This can be reproduced with default setting.

**Comment by siduck** [comment]
not a nvchad issue, either edit our telescope config or just use the default theme

**Comment by MKrbm** [comment]
So isn't it bult-in feature of NavChad to modify UI of telescope? 
I thought NvChad/ui handle them.

**Comment by siduck** [comment]
> So isn't it bult-in feature of NavChad to modify UI of telescope? I thought NvChad/ui handle them.

base46 adds highlights for telescope UI elements. the searchbar shouldnt look like that tho. try the bordered theme for telescope in your chadrc

**Comment by MKrbm** [comment]
Indeed I found if I overwrite plugins directory as below, it turns out to be alright.

`integrations\telescope.lua\
![image](https://github.com/NvChad/NvChad/assets/57752203/dd00c2c4-5a7b-4600-88b4-9c3a8fea0841)

`customs\configs\telescope.lua`
![image](https://github.com/NvChad/NvChad/assets/57752203/520afc11-e93a-406b-b07f-1a7d677d146a)



The problem I found instead is that I cannot overwrite the hl defined in `lua\base46\integrations\telescope.lua` from my `custom` folder maybe due to caching.
Here, I was trying to set "None" to `TelescopePromptPrefix` but it isn't applied ( If I set something else not listed in first image, then it will be applied)

So I'd like to know How to overwrite the variables defined in base46 from config files.
